### PR TITLE
Capsule/mask classifier and Capsule/classifier_vehicle_color_openvino improvements

### PR DESCRIPTION
**Pull request by apockill**

### CODE REVIEW

File: capsules/segmenter_person_openvino/licenses/model.license [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (velovix) [inline capsules/segmenter_person_openvino/licenses/model.license]: Duplicate license file

File: capsules/segmenter_person_openvino/capsule.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from vcap import (
```
Comment (velovix) [inline capsules/segmenter_person_openvino/capsule.py]: Probably segmenter_person_openvino is not supposed to be in this PR.

**Review by velovix — APPROVED** [review]

File: capsules/classifier_mask_closeup/meta.conf [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[about]
+api_compatibility_version = 0.2
```
Comment (BryceBeagle) [inline capsules/classifier_mask_closeup/meta.conf]: newline

**Review by BryceBeagle — CHANGES_REQUESTED** [review]
This PR includes changes from other capsules. Was this intentional?

File: capsules/classifier_mask_closeup/meta.conf (hunk 3 above)
Comment (apockill) [reply capsules/classifier_mask_closeup/meta.conf]: Fixed

**Comment by apockill** [comment]
@BryceBeagle initially I didn't mean to add them, but then I realized I'd committed some improvements that I'd intended to push anyways, and so I fit them in this review. 

It basically fixed a bug with the vehicle classifier where the label "vehicle" was supposed to be "car", according to the model. All the truck/bus predictions were incorrect because of this bug.

**Comment by BryceBeagle** [comment]
I'd suggest changing the PR name, then

**Pull request merged**